### PR TITLE
NMS-13929: Add a note on consolidate features

### PR DESCRIPTION
**Pull request by cgorantla**
### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-13929

### CODE REVIEW

**Review by Bonrob2 — COMMENTED** [review]
Minor changes and a question.

File: docs/modules/releasenotes/pages/whatsnew.adoc [hunk 1]
```diff
@@ -57,6 +57,9 @@ Additional settings are needed on both Minion and {page-component-title} to enab
 
 Refer to the deployment section in the documentation for details.
 
+NOTE : 29.0.5 consolidated IPC features into one feature on Minion. This groups all IPC features into either `openms-core-ipc-jms` or `opennms-core-ipc-kafka`.
```
[the remark this answers is not in the document]
Comment (cgorantla) [reply docs/modules/releasenotes/pages/whatsnew.adoc]: Updated. Let me know if that clarifies things.

**Review by Bonrob2 — APPROVED** [review]
LGTM!

File: docs/modules/releasenotes/pages/whatsnew.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (Bonrob2) [reply docs/modules/releasenotes/pages/whatsnew.adoc]: Yes, thanks. This makes more sense to me.

**Review by RangerRick — APPROVED** [review]

**Pull request merged**